README.md

```diff
@@ -1 +1,89 @@
-# stacks-dotnet-packages-sqs
+# Amido Stacks Messaging AWS SQS
+
+This library is wrapper around AWS SQS.
+The main goal is:
+
+    1.) to publish messages to a configured SQS queue
+    2.) to receive messages from a configured SQS queue
+
+## 1. Requirements
+
+You need a SQS instance in order to use this library. You can follow the official instructions provided by AWS [here](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/sqs-apis-intro.html).
+
+## 2. Registration/Usage
+
+### 2.1 Dependencies
+- `Amido.Stacks.Configuration`
+- `AWSSDK.SQS`
+
+### 2.2 Currently Supported messages
+
+The library currently supports:
+- publishing and receiving events implementing `Amido.Stacks.Application.CQRS.ApplicationEvents.IApplicationEvent`
+
+### 2.3 Usage in dotnet core application
+
+#### 2.3.1 Event
+In this case the `NotifyEvent` has a `NotifyEventHandler`. The handler implements
+`Amido.Stacks.Application.CQRS.ApplicationEvents.IApplicationEventHandler<NotifyCommand, bool>` and the command implements
+`Amido.Stacks.Application.CQRS.ApplicationEvents.IApplicationEvent` interfaces.
+
+***NotifyEvent.cs***
+
+```cs
+   public class NotifyEvent : IApplicationEvent
```

Comment:
Can we use an event we have in the app like `MenuCreated`? The `NotifyEvent` is from the ASB package.

Reply:
Yeah, that makes sense. Thanks :)